code/modules/departments/department.dm

```diff
@@ -1,15 +1,24 @@
-/datum/department
-	var/title = "Undefined"         // Player facing. Can be changed freely without breaking code or updating refrences in jobs.
-	var/reference = "undefined"     // Code facing. Jobs reference their department by this.
+/proc/cmp_departments_dsc(var/decl/department/a, var/decl/department/b)
+	. = b.display_priority - a.display_priority
+
+/decl/department
+	var/name                        // Player facing. Can be changed freely without breaking code or updating refrences in jobs.
+	var/reference                   // Code facing. Jobs reference their department by this.
```

Comment:
I think this is redundant, jobs should just refere to their dept with type, would be able to just get department via decl procs rather than job controller having their own version.